example/src/App.js

```diff
@@ -6,8 +6,9 @@ const auth = new Client()
 
 function App() {
   let [session, setSession] = useState(null)
-  let [email, setEmail] = useState('')
+  let [email, setEmail] = useState(localStorage.getItem('email') ?? '')
   let [password, setPassword] = useState('')
+  let [remember_me, setRememberMe] = useState(false)
```

Comment:
One small request on style - can we please make this camel case? `rememberMe`

Reply:
ok

Reply:
@kiwicopple I have change this variable's name, Please check it out.